voxpupuli-release.gemspec

```diff
@@ -19,4 +19,5 @@ Gem::Specification.new do |s|
   # Runtime dependencies, but also probably dependencies of requiring projects
   s.add_runtime_dependency 'rake'
   s.add_runtime_dependency 'puppet-blacksmith', '>= 4.0.0'
+  s.add_development_dependency 'github_changelog_generator'
```

Comment:
This is the very old released version from 2016. Is that new enough?

Reply:
I tested it and it worked fine in this repository.